<?xml version="1.0" encoding="UTF-8"?> 
<launch>
  <group ns="rosbot1">
    <node name="gt_to_path" pkg="tf_to_trajectory" type="gazebo_gt_to_path.py" output="screen">
        <param name="parent_frame_id" value="world"/>
        <param name="append_rate" value="3"/>
    </node>
  </group>
</launch>
